atef/widgets/config/page.py

```diff
@@ -1659,3 +1659,16 @@ def clean_up_any_comparison(self) -> None:
         - Cleans up all the sub-nodes.
         """
         self.tree_item.takeChildren()
+
+
+class ProcedureGroupPage(ConfigurationGroupPage):
```

Comment:
Is this used at all yet?

Reply:
Nope, This is probably left over from when my goal was to do active checkouts, before I pared down the scope of this PR.